prisma/schema.prisma

```diff
@@ -13,4 +13,9 @@ datasource db {
 model User {
   id    String @id @default(uuid())
   email String @unique
+
```

Comment:
Why separate fields by a line?

Reply:
Just to separate the ones that are editable by the user from the ones that are automatically managed by the prisma.

Reply:
Following this logic, the `id` should be separated too, right?

Reply:
Really, I'm going to edit the way it would be in the order of the database and this separation by spaces, if want I can remove the spaces and leave everything together.

Reply:
I think removing spaces is better.